docs/rest_api.md

```diff
@@ -1251,3 +1251,13 @@ Remove a user from a project.
 Authorization requirements:
 
 * Administrative access.
+
+#### list_active_extensions
+
+`GET /active_extensions`
+
+Request Body:
+
+{}
+
+List all active extensions.
```

Comment:
You should add a bit about the authorization requirements and how the response body looks like.